code/datums/status_effects/debuffs.dm

```diff
@@ -674,6 +674,9 @@
 	. = ..()
 	REMOVE_TRAIT(owner, TRAIT_MUTE, id)
 
+/datum/status_effect/transient/silence/absolute
+	id = "abssilenced"
```

Comment:
You should probably add a comment here explaining how this differs from regular silence (noting that it specifically only mutes emotes).

Reply:
Done